LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018-2022 Safe Ecosystem Foundation
```

Comment:
Is this correct seeing as the foundation is new, or should it reference Gnosis first?

Reply:
It's the license from safe-react, have you read the PR description?

Reply:
It slipped my mind that we adjusted `safe-react` directly to the foundation.